rfcs/text/0000-dss-deletion-process.md

```diff
@@ -0,0 +1,416 @@
+### DCP PR:
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC Deletion of data in the DCP
+
+## Summary
+Establish a secure and controlled process for deleting data from the Data Store.
+
+
+## Author(s)
+
+ [Trent Smith](mailto:trent.smith@chanzuckerberg.com)
+ 
+ [Hannes Schmidt](mailto:hannes@ucsc.edu)
+
+## Shepherd
+ [Rhian Anthony](mailto:ranthony@broadinstitute.org)
+
+## Motivation
+
+One of the DCP's primary objectives is to ensure the durability of the
+(meta)data it contains and to maintain a complete record of the changes made to
+it. The DCP achieves this by never overwriting a (meta)data file but instead
```

Comment:
AFAIK this is not a primary objective of the DCP - please provide a reference to this requirement if you have one. 
I think it _may_ be a design goal of the DSS, but is more likely just a simplifying assumption for the DSS implementation.

Reply:
Do you have a recommendation or would it be better to remove all together?

Reply:
@sampierson while not a "primary objective", I think it's transitively a stated shared goal of the DCP because the DSS owns the responsibility and process around storing and providing access to data (https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/DataStore/charter.md). In order to fulfill this responsibility, in particular in order to provide both data revision ("AUDR") capabilities and stable (immutable) release URI capabilities, the DSS implements an immutable versioned storage model.